tests/beaker_tests/netdev_stdlib/name_server_provider_defaults.rb

```diff
@@ -74,7 +74,7 @@
 
     # Expected exit_code is 0 since this is a vegas shell cmd.
     # Flag is set to true to check for absence of RegExp pattern in stdout.
-    cmd_str = UtilityLib.get_vshell_cmd('show running-config section name-server')
+    cmd_str = UtilityLib.get_vshell_cmd('show running-config | section name-server')
```

Comment:
I thought of that but decided to keep it because it is the switch setup test case.